onebusaway-android/src/androidTest/java/org/onebusaway/android/util/test/UIUtilTest.java

```diff
@@ -554,6 +556,20 @@ public void testArrivalInfoLabels() {
         assertEquals("On time", arrivalInfo.get(31).getStatusText());
     }
 
+    public void testMaybeShrinkRouteName() {
+        TextView tv = new TextView(getContext());
+
+        String routeShortName = "Test";
```

Comment:
Could you also please add one more test here with a route name with 3 characters, to verify that the TextView text size doesn't change?  You could set it to a hard-coded value here before the test just for a simple test.